src/js/ui/analyse/AnalyseCtrl.ts

```diff
@@ -207,7 +207,7 @@ export default class AnalyseCtrl {
       orientation: this.vm.flip ? oppositeColor(this.orientation) : this.orientation,
       movableColor: this.gameOver() ? null : color,
       dests: dests || {},
-      check: node.check,
+      check: node.check !== undefined ? node.check : false,
```

Comment:
is it equivalent to `check: !!node.check`?

Reply:
yes :) I should change that